frontend/views/components/CreateGroupSteps/GroupMincome.vue

```diff
@@ -56,16 +56,16 @@ export default {
   },
   methods: {
     update (e) {
-      this.v[e.target.name].$touch()
+      this.vForm[e.target.name].$touch()
```

Comment:
> This PR is somewhat cosmetic, it improves the situation described in #661 slightly. I left a couple of comments. Note that the tests aren't passing.
> 
> It would be great if this could do a more thorough job of closing #661, by investigating whether it's possible to delete some of the current spaghetti trampoline code (i.e. all the event emission code) , and the still-confusing, possibly unnecessary dichotomy between `vForm` and `$v`.

I understand the issue here, but I'm not a master with vuelidate to come with a better answer. I can try to take a look again later but maybe @pieer has a better idea?